samples/calculator/README.md

```diff
@@ -13,18 +13,26 @@ Common Kotlin module contains arithmetic expressions parser.
 ## Android App
 The common module may be used in an Android application.
 
-To build and run the Android sample do the following:
+Please make sure that Android SDK version 28 is installed, using Android SDK manager in Android Studio.
+See https://developer.android.com/studio/index.html for more details on Android Studio or
+`$ANDROID_HOME/tools/bin/sdkmanager "platforms;android-28" "build-tools;28.0.3"` from command line.
 
-1.  Open the project in Android Studio 3.1
-2.  Create a new Android App configuration. Choose module `android`.
-3.  Now build and run the configuration created.
+To build use `ANDROID_HOME=<your path to android sdk> ../gradlew assemble`.
+
+Run `$ANDROID_HOME/platform-tools/adb install -r androidApp/build/outputs/apk/debug/androidApp-debug.apk`
+to deploy the apk on the Android device or emulator (note that only ARM-based devices are currently supported).
```

Comment:
> note that only ARM-based devices are currently supported

This doesn't look correct to me.

Reply:
Yes. I will remove the statement about ARM-based devices.